image/user/boot.json

```diff
@@ -3,12 +3,12 @@
         "slots": {
             "a": {
                 "bootable": true,
-                "successful": false,
+                "successful": true,
```

Comment:
IMO it should be set to false - first successful boot will make this `true` - in case of bootloop after flashing

Reply:
That depends: if we assume that there is a possibility we ship broken image to the factory, then yes, you're right.

Reply:
Let's revert this change as the previous version provides a bit more safety.